routes/organisation/steps.js

```diff
@@ -9,12 +9,12 @@ const DeleteUserController = require('../../controllers/UserManagement/DeleteUse
 const UserEditController = require('../../controllers/UserManagement/UserEditController');
 const ResetUserPasswordController = require('../../controllers/UserManagement/ResetUserPasswordController');
 const UserCreateController = require('../../controllers/UserManagement/UserCreateController');
+const OrganisationCreateController = require('../../controllers/UserManagement/OrganisationCreateController');
 
 const externalRoutes = {
   '/': {
     controller: OrganisationsController,
     entryPoint: true,
-    resetJourney: true,
```

Comment:
Given this page didn't really do much interactive previously, I don't think that removing this has any real implications